plugins/catalog-backend/src/permissions/rules/isEntityKind.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Entity } from '@backstage/catalog-model';
+import { EntitiesSearchFilter } from '../../catalog/types';
+
+/**
+ * A {@link CatalogPermissionRule} which filters for entities with a specified
+ * kind.
+ * @public
+ */
+export const isEntityKind = {
```

Comment:
should this one (and the others) perhaps actually say `: CatalogPermissionRule` then? It'll catch typescript errors nearer to the actual source.